herbridge/frontend/src/components/App/App.js

```diff
@@ -29,7 +29,48 @@ const theme = createMuiTheme({
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {}
+    this.state = {
+      isLoggedIn: false,
+      loginError: null,
+      loginIsLoading: false
```

Comment:
pretty minor, but let's do trailing commas on collections so that future diffs will be cleaner